spring-boot/admin/src/test/resources/bootstrap.properties

```diff
@@ -0,0 +1,8 @@
+spring.application.name=ygg-admin
+spring.cloud.config.uri=https://config.coatrack.eu
```

Comment:
To make it easier understandable for the reader, maybe we should make clear that these are dummy values for the test. E.g.

```suggestion
spring.cloud.config.uri=https://test-dummy-config-url.coatrack.eu
```

Reply:
Done, and it worked well